index.js

```diff
@@ -38,7 +56,7 @@ function toMajor (mcVersion, preNetty, typeArg) {
   const type = typeArg || (parts.length === 2 ? parts[0] : 'pc')
   const version = parts.length === 2 ? parts[1] : mcVersion
   let majorVersion
-  if (data[type][version]) { majorVersion = version } else if (versionsByMinecraftVersion[type][version]) { majorVersion = versionsByMinecraftVersion[type][version].majorVersion } else if (preNetty && preNettyVersionsByProtocolVersion[type][version]) { return toMajor(preNettyVersionsByProtocolVersion[type][version][0].minecraftVersion, preNetty, type) } else if (!preNetty && postNettyVersionsByProtocolVersion[type][version]) { return toMajor(postNettyVersionsByProtocolVersion[type][version][0].minecraftVersion, preNetty, type) } else if (versionsByMajorVersion[type][version]) { majorVersion = versionsByMajorVersion[type][version].minecraftVersion }
+  if (loadData(type, version)) { majorVersion = version } else if (versionsByMinecraftVersion[type][version]) { majorVersion = versionsByMinecraftVersion[type][version].majorVersion } else if (preNetty && preNettyVersionsByProtocolVersion[type][version]) { return toMajor(preNettyVersionsByProtocolVersion[type][version][0].minecraftVersion, preNetty, type) } else if (!preNetty && postNettyVersionsByProtocolVersion[type][version]) { return toMajor(postNettyVersionsByProtocolVersion[type][version][0].minecraftVersion, preNetty, type) } else if (versionsByMajorVersion[type][version]) { majorVersion = versionsByMajorVersion[type][version].minecraftVersion }
```

Comment:
May I simply ask... what the heck is going on at this line?

Reply:
I have no idea. I only change `data[...][...]` to `data(..., ...)` at the beginning.

Reply:
Bare minimum formatting:
```js
if (loadData(type, version)) {
  majorVersion = version
} else if (versionsByMinecraftVersion[type][version]) {
  majorVersion = versionsByMinecraftVersion[type][version].majorVersion
} else if (preNetty && preNettyVersionsByProtocolVersion[type][version]) {
  return toMajor(preNettyVersionsByProtocolVersion[type][version][0].minecraftVersion, preNetty, type)
} else if (!preNetty && postNettyVersionsByProtocolVersion[type][version]) {
  return toMajor(postNettyVersionsByProtocolVersion[type][version][0].minecraftVersion, preNetty, type)
} else if (versionsByMajorVersion[type][version]) {
  majorVersion = versionsByMajorVersion[type][version].minecraftVersion
}
```

Reply:
Wonder why it was compressed into a single line?

Reply:
Idk. Made a quick commit to format it.